app/org/sagebionetworks/bridge/models/ClientInfo.java

```diff
@@ -134,6 +134,18 @@ public Integer getSdkVersion() {
         return sdkVersion;
     }
     
+    public boolean isSupportedAppVersion(Integer minSupportedVersion) {
```

Comment:
This was a Long in the study. I think you want it to be consistently Integer or Long.